.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,175 @@
+# This workflow is the main workflow for regenerating the benchmarks data needed for Bowtie's UI.
+# It runs all benchmarks over Bowtie's supported implementations, publishing the benchmark reports for use in the frontend.
+name: Collect New Benchmark Results
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Monday at 08:00 UTC
+    - cron: "0 8 * * 1"
```

Comment:
Running at 0 on the hour is not smart for similar reasons -- you need to imagine the distribution of jobs over the day -- it's going to be very high at midnight, and then have little bumps on the hour every hour, and then littler bumps every hour on multiples of 10 or 5, or etc.

But OK, we can leave this.

Reply:
Anyways our benchmarks are gonna require about 2.5 hrs from what I have tested, so thats why this shouldnt make much of a difference.